backend/src/tasks/hibp.ts

```diff
@@ -78,12 +88,16 @@ export const handler = async (commandOptions: CommandOptions) => {
           state: 'open',
           source: 'hibp',
           needsPopulation: false,
-          structuredData: { emails: results },
+          structuredData: {
+            emails: results['Emails'],
+            breaches: results['Breaches']
+          },
           description: `Emails associated with ${domain.name} have been exposed in a breach.`
         })
       );
       await saveVulnerabilitiesToDb(vulns, false);
-      console.log(results);
+      console.log(results['Emails']);
```

Comment:
Can you actually remove these console.log statements? This may clog up the logs with too much data